### PR TITLE
chore: added Stalebot

### PR DESCRIPTION
**Pull request by mxschmitt**
Might be a good idea to get rid of inactive issues.

### CODE REVIEW

File: .github/stale.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+# Issues with these labels will never be considered stale
+exemptLabels:
+  - pinned
+  - security
+# Label to use when marking an issue as stale
+staleLabel: wontfix
+# Comment to post when marking an issue as stale. Set to `false` to disable
+markComment: >
+  This issue has been automatically marked as stale because it has not had
+  recent activity. It will be closed if no further activity occurs. Thank you
```
Comment (bpas247) [inline .github/stale.yml]: Should we include how long before it will close if no further activity occurs?

File: .github/stale.yml [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# Number of days of inactivity before an issue becomes stale
+daysUntilStale: 60
+# Number of days of inactivity before a stale issue is closed
+daysUntilClose: 7
+# Issues with these labels will never be considered stale
+exemptLabels:
+  - pinned
+  - security
```
Comment (bpas247) [inline .github/stale.yml]: I think we should create a new label that will be used to mark PRs that are currently blocked (waiting on dependency updates, waiting on other PRs to merge first, etc).

File: .github/stale.yml [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+# Label to use when marking an issue as stale
+staleLabel: wontfix
+# Comment to post when marking an issue as stale. Set to `false` to disable
+markComment: >
+  This issue has been automatically marked as stale because it has not had
+  recent activity. It will be closed if no further activity occurs. Thank you
+  for your contributions.
+# Comment to post when closing a stale issue. Set to `false` to disable
+closeComment: false
```
Comment (bpas247) [inline .github/stale.yml]: Do we want to not mark issues as stale? We have quite a few issues that are pretty outdated atm.

**Comment by taion** [comment]
Are there reasonable defaults here, or do we need to explicitly configure all the bot values?

Also should we take a quick audit and pin long-standing old issues before updating?

File: .github/stale.yml (hunk 1 above)
Comment (mxschmitt) [reply .github/stale.yml]: makes sense

**Comment by krichter722** [comment]
This is a bad idea on all imaginable projects. You'll have to triage manually one day in order to figure out which issues are active and which are not. The assumptions that issues go away is just not true even if stale and similar bots support this line of thinking.

**Pull request closed without merging**